app/src/app/(payload)/custom.scss

```
@@ -579,6 +579,27 @@ body {
     color: var(--theme-elevation-1000);
   }
 
+  // The two block labels inside a subject-grade group — "Subject Administrator" and
+  // "Editing access (N)" — added 2026-08-19 after an operator misread the administrator's row as a
+  // column header for the editor rows beneath it. Naming both blocks is the fix; this rule is what
+  // makes a label look like one.
+  //
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)